content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+== WebAuthn primer for mobile development
+This section should be referenced if you are looking to begin development for a mobile application, but are unfamiliar with WebAuthn concepts and development. The goal of this page is threefold:
+
+* Point you in the direction of  resources to learn more about the WebAuthn protocol
+* Provide additional considerations for developing WebAuthn on mobile devices
```

Comment:
Consider rephrasing to be more focused and direct. Something like:
* Design considerations for mobile apps
** platform vs cross-platform
** trusted devices
** hardware-backed vs copyable
** etc...

Reply:
Changed in newest commit